src/main/java/ru/sbtqa/tag/apifactory/rest/RestEntityImpl.java

```diff
@@ -119,11 +119,6 @@ public Bullet post(String url, Map<String, String> headers, Object body) throws
 
             HttpResponse response = client.execute(post);
 
-            if (response.getStatusLine().getStatusCode() != 200) {
```

Comment:
Why you delete status code checking in POST and don't do it in GET?
https://github.com/Filter94/api-factory/blob/fd7700471e9babd43109dd928af10b8fbfd402f9/src/main/java/ru/sbtqa/tag/apifactory/rest/RestEntityImpl.java#L70

Reply:
Removed.

Reply:
Now it's ok. I'll release new version for you